libs/small-steps/small-steps.cabal

```
@@ -40,6 +40,7 @@ library
   build-depends:
     aeson,
     base >=4.18 && <5,
+    cardano-data,
```

This needs a lower bound
```suggestion
    cardano-data >=1.3,
```